dimensions/overworld.json

```diff
@@ -295,10 +349,10 @@
         },
         {
             "minHeight": 2,
```

Comment:
The Diamond Changes Cant be at 0 per chunk, otherwise there is not enough ore with this cave generation